app/views/embassies/index.html.erb

```diff
@@ -18,22 +18,19 @@
       <% @embassies_by_location.each do |embassy| -%>
       <li>
         <h2><%= embassy.name %></h2>
-        <% if embassy.offices.empty? -%>
-          <p><%= embassy.name %> doesn't have consular services, you can get consular assistance from the local authorities.</p>
+
+        <% if !embassy.has_no_consular_service_in_location? && embassy.offices.empty? -%>
```

Comment:
Minor point... I try to avoid negating negative things as it hurts my head. `embassy.has_consular_service_in_location?` and then `!embassy.has_consular_service_in_location?` on line 26 reads more clearly to me.